ios/autotools-ios-helper.sh

```diff
@@ -63,8 +63,8 @@ build_iphoneos() {
     PLATFORM="iphoneos"
     ARCH_OPTS="--enable-armasm --enable-sp-asm"
     ARCH_FLAGS="-arch arm64"
-    HOST_FLAGS="${ARCH_FLAGS} -mfpu=auto -miphoneos-version-min=${MIN_IOS_VERSION} -isysroot $(xcrun --sdk ${SDK} --show-sdk-path)"
-    CHOST="aarch64-apple-darwin"
+    HOST_FLAGS="${ARCH_FLAGS} -miphoneos-version-min=${MIN_IOS_VERSION} -isysroot $(xcrun --sdk ${SDK} --show-sdk-path)"
+    CHOST="arm64-apple-ios"
```

Comment:
I see. So the change in https://github.com/expressvpn/lightway-core/pull/46/files#diff-8fe0db5287c1966902957184569fd42998a2b00ce09d2f2e69562423f6c8185fR67 breaks it on older iPhones, now we are changing it back. Could you be more specific on exactly which "iPhone model" is "old"?

Reply:
Ahh the iphone I tested here was iPhone 6S, this was broken. 

Its not exactly the same its `arm64-apple-ios` instead of `arm-apple-darwin`

Reply:
I see. But what's the difference between `aarch64-apple-ios` and `arm64-apple-darwin`? Why using `arm64-apple-darwin` fix the issue on iPhone 6S?

Also, are we sure that using `arm64` instead of `arm` doesn't break it on even older iOS devices?

Reply:
I'm not 100% certain what the issue is here, but I'm thinking its due to differences in the `-darwin` vs `-ios` target. The LLVM code https://github.com/llvm/llvm-project/blob/main/llvm/lib/Support/Triple.cpp seems to indicate its different

All devices supported by our deployment target iOS12 is actually an arm64 device. We also set the arch to arm64.